lib/libutee/default_bold.c

```diff
@@ -0,0 +1,1009 @@
+/*
```

Comment:
The python script depends on some additional packages and you'll only need to regenerate the fonts if you change resolution etc. There's pros and con in both ways.